### ISSUE TITLE
Menu clicks in dropdown does not works

### ISSUE THREAD

**Issue by Pitel** [jwstegemann/fritz2#463]
**Describe the bug**
I have a menu inside dropdown. Click events on menu items does not work.

**To Reproduce**
1. https://components.fritz2.dev/#Menu
2. Scroll down to _Dropdown Menus_.
3. Click on the dropdown hamburger icon, the menu appears.
4. Click on _Clickable entry_.
5. Based on the menu above, toast should appear, but nothing happens!

**Expected behavior**
Toast appears

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Chrome
 - Version 93

**Comment by chausknecht** [comment]
Insteresting!

With a Fierfox (89.0.2, with Windows 10) I can't reproduce this behaviour:
![grafik](https://user-images.githubusercontent.com/70946811/123756268-0a68f000-d8bd-11eb-88ed-b4467fcfcadc.png)

With a Chrome (Version 91.0.4472.124, with Windows 10) I can't reproduce this either:
![grafik](https://user-images.githubusercontent.com/70946811/123756974-c62a1f80-d8bd-11eb-8ae8-bef72e5f8924.png)

Have you tried to view the DOM and whether the toast gets rendered?

We have one cdev in our team that also runs Linux for development - hopefully he can try to reproduce the bug with a seeting near to yours!

(I do not like the `compact` setting for the toasts; they appear to small and not eye catching enough; we should definitly change that to the default toast variant)

**Comment by Pitel** [comment]
It gets even weirder. When using mouse, it works realiably, toast is displayed. But when using my laptop's touchpad, it almost never works when I just tap on the touchpad. But when I press the touchpad (so it makes click sound), it works half the time. :scream_cat:

But the menu itself (outside dropdown) works reliably every time, also any other links anywehere on the web. So I suppose this might be some weird race condition in dropdown events handling. I'm also realy curious what might be so different between tap and click on touchpad as JS events...

Note that the Chrome 93 is canary, so there might be something broken here. But I've also tested in in latest Firefox, and it behaves the same.

**Comment by chausknecht** [comment]
Could you please test this with the stable browsers versions too?

Touch events often differ from mouse events. So we lack explicit touch support in many components (slider has a known insufficiency for example) - it is possible, that "simulating" a click with the touchpad creates a different event than `clicks`.

**Comment by haukesomm** [comment]
> I do not like the compact setting for the toasts; they appear to small and not eye catching enough; we should definitly change that to the default toast variant

@chausknecht I opened a PR that fixes the alert-toasts size in kitchensink, it must have been a leftover from the recent refactorings. See [#68](https://github.com/jwstegemann/fritz2-kitchensink/pull/68) (fritz2-kitchensink) ✌🏻

**Comment by Pitel** [comment]
The Firefox was stable (89.0.2). And tested with stable Chrome (91.0.4472.114) and it's the same. So I don't think it;s that much browser dependent.

**Comment by chausknecht** [comment]
@Pitel Did you check the DOM? If the timeout is too short, you could build the KS by youreself and change the timeout.

**Comment by Pitel** [comment]
@chausknecht I've checked the DOM now, and it's consistent with shat is displayed (or not). If I just tap on my touchbar, DOM is not changed.

**Comment by chausknecht** [comment]
But is the Toast rendered if you click on the menu? (with real click!)

**Comment by Pitel** [comment]
Yes. If do a real click (with mouse), the toast is added to a DOM and rendered on screen. If I just tap on tcouhpad, DOM is unchaged (well, the dropdown disappear, but no change regarding toasts) and no toast is rendered on screen.

**Comment by chausknecht** [comment]
I am quite unshure, whether this is a problem with our framework or rather with your touchpad: Afaik physical "clicking" on a touch device should be mapped to mouse clicks. If that does not happen, there is no event we could rely on.

Can you please test, whether you can click with the same touch onto our button components?

**Comment by Pitel** [comment]
Yes. Clicking (even without the "click" noise) on buttons in Buttons Kithcen Sink page works realiably. Toasts are displayed, Modals are opened (and closed with clicking on the close button).

**Comment by chausknecht** [comment]
Ok that's really weird... :-(

**Comment by jamowei** [comment]
Hi, I take a deep dive into this problem and found a possible solution to fix this issue. Can you please try to reproduce your problem at this version of our kitchensink demo: https://jamowei.github.io/kitchensink-bug/index.html

Thanks.

**Comment by Pitel** [comment]
@jamowei I tired it, and behaves pretty much the same. I didn't noticed any difference. :(

**Comment by Pitel** [comment]
@jamowei It behaves pretty much the same: "tapping" on the clickable entry in dropdown with touchpad just hides the dropdown and won't display the toast. But in some cases, the toast is displayed. It's pretty much random, but not working most of the times.
